windows-driver-docs-pr/install/combining-platform-extensions-with-operating-system-versions.md

```diff
@@ -99,6 +99,25 @@ When you install a driver package on a device, Windows follows these steps to pr
 
 1. If Windows cannot find a matching [**INF *Models* section**](inf-models-section.md), it will not install that driver package on the device.
 
+### How *TargetOsVersion* decorations are resolved
+
+When determining the applicable [**INF *Models* sections**](inf-models-section.md), the evaluation is performed per-line within the [**INF Manufacturer section**](inf-manufacturer-section.md), where, for a given OS version, each line may have up to one applicable section.
+
+The below INF excerpt shows an example where [ExampleModelsSection_1.NTamd64.10.0...17134] will be selected on a Windows 10 build equal to or greater than 17134, and *both* [ExampleModelsSection_1.NTamd64.10.0...17134] and [ExampleModelsSection_2.NTamd64.10.0...22000] will be selected on a Windows 10 build equal to or greater than 22000.
```

Comment:
Build 22000 is a Win11 build, so "Windows 10 build equal to or greater than 22000." may be misleading.  Should this be simplified to "will be selected on a Windows build equal to or greater than 22000"?